src/iop/overlay.c

```diff
@@ -160,7 +160,7 @@ const char **description(struct dt_iop_module_t *self)
 
 const char *aliases()
 {
-  return _("composition|layering|stacked");
+  return _("composition|layering|stacked|overlay");
```

Comment:
"Composition" is not the correct term here. You could put composite and compositing but composition is wrong

Reply:
I would still prefer to make the tenses of the others consistent. So overlay/layer/stack.